vnext/Scripts/OpenSSL.nuspec

```diff
@@ -7,20 +7,33 @@
     <authors>Microsoft</authors>
     <projectUrl>https://www.openssl.org</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
```

Comment:
is this right?

Reply:
The URL is. I do not know about `requireLicenseAcceptance`.